corehq/form_processor/parsers/form.py

```diff
@@ -165,7 +167,16 @@ def _handle_duplicate(new_doc):
     """
     interface = FormProcessorInterface(new_doc.domain)
     conflict_id = new_doc.form_id
-    existing_doc = FormAccessors(new_doc.domain).get_with_attachments(conflict_id)
+    try:
+        existing_doc = FormAccessors(new_doc.domain).get_with_attachments(conflict_id)
+    except ResourceNotFound:
```

Comment:
Is it possible that some issue other than missing attachments would cause this error? It's a generic couchdbkit error that could be raised if the doc is not found, right? Does that matter?